compilation/abiutils/solidity_errors.go

```diff
@@ -105,3 +105,35 @@ func GetSolidityCustomRevertError(contractAbi *abi.ABI, returnError error, retur
 	}
 	return nil, nil
 }
+
+// GetPanicReason will take in a panic code as an uint64 and will return the string reason behind that panic code. For
+// example, if panic code is PanicCodeAssertFailed, then "assertion failure" is returned. This is used by the execution
+// tracer to print out the panic reason.
+func GetPanicReason(panicCode uint64) string {
+	// Switch on panic code
+	switch panicCode {
+	case PanicCodeCompilerInserted:
+		return "compiler inserted panic"
+	case PanicCodeAssertFailed:
+		return "assertion failed"
+	case PanicCodeArithmeticUnderOverflow:
+		return "arithmetic underflow"
+	case PanicCodeDivideByZero:
+		return "division by zero"
+	case PanicCodeEnumTypeConversionOutOfBounds:
+		return "enum access out of bounds"
+	case PanicCodeIncorrectStorageAccess:
+		return "incorrect storage access"
+	case PanicCodePopEmptyArray:
+		return "pop on empty array"
+	case PanicCodeOutOfBoundsArrayAccess:
+		return "out of bounds array access"
+	case PanicCodeAllocateTooMuchMemory:
+		return "overallocation of memory"
+	case PanicCodeCallUninitializedVariable:
+		return "call on uninitialized variable"
+	default:
+		// TODO: Should we throw a panic here or just return this string?
```

Comment:
I'd say here we can return a string that is more like "unknown panic code(<code>)", so its still clear what it is.

Reply:
Resolved in [cd27dcb](https://github.com/crytic/medusa/pull/161/commits/cd27dcb384e4f0ae2dc52f67534a6b329b1c1ec1)